notebooks/en/code_search.ipynb

```diff
@@ -0,0 +1,779 @@
+{
```

Comment:
<p>Same comment for this NOTE here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>